source/uwp/UWPUITestLibrary/UITest.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace UWPUITestLibrary
+{
+    [TestClass]
+    public class UITest
+    {
+        protected static Application application = null;
+
+        [ClassInitialize]
+        public static void Setup(TestContext testContext)
+        {
+            application = new Application();
+            application.Initialize();
```

Comment:
@almedina-ms looks good, but I'm unable to get the tests to run successfully (locally or in CI) from this branch.

I get an error at ^this line (`application.Initialize()`):

```
Class Initialization method UWPUITestLibrary.UITest.Setup threw exception. System.Exception: System.Exception: Failed to find 'AdaptiveCardsUWPUITestApp*.appx' or '*AdaptiveCardsUWPUITestApp.msix' in [...]source\uwp\UWPUITestApp'
```